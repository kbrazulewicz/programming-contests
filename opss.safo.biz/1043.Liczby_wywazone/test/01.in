5
1239999
13000
90865
189999
79878999
