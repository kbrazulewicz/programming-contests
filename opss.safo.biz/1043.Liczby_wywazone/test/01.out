1240002
13002
90866
190002
79879002
